views/js/pciCreator/ims/mathEntryInteraction/runtime/mathEntryInteraction.js

```diff
@@ -969,17 +1128,21 @@ define([
                     pciInstance.off('latexGapInput');
 
                     mathEntryInteraction.destroy();
+                },
+
+                getPropertyManager() {
```

Comment:
**chore:** The name `propertyManager` was given as an example, I think a better name should be set. Something like `getInputs()` or something else.